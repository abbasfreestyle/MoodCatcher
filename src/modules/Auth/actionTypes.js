import { keyMirror } from 'utils/keyMirror';
import { addNameToActionTypes } from 'utils/addNameToActionTypes';

import { NAME } from './name';

const actionTypes = ['SIGNED_IN', 'SIGNED_OUT'];

export default addNameToActionTypes(NAME, keyMirror(actionTypes));
