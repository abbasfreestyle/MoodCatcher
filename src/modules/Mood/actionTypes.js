import { keyMirror } from 'utils/keyMirror';
import { addNameToActionTypes } from 'utils/addNameToActionTypes';

import { NAME } from './constants';

const actionTypes = [
  'SET_MOOD',
  'ADD_FEELING',
  'REMOVE_FEELING',
  'UPDATE_COMMENT',
  'RESET_EVERYTHING'
];

export default addNameToActionTypes(NAME, keyMirror(actionTypes));
