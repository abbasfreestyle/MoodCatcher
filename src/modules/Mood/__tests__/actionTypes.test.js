import actionTypes from '../actionTypes';

describe('Mood > actionTypes snapshot', () => {
  test('should match', () => {
    expect(actionTypes).toMatchSnapshot();
  });
});
