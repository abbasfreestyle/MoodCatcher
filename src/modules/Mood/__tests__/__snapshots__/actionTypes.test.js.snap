// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mood > actionTypes snapshot should match 1`] = `
Object {
  "ADD_FEELING": "Mood/ADD_FEELING",
  "REMOVE_FEELING": "Mood/REMOVE_FEELING",
  "RESET_EVERYTHING": "Mood/RESET_EVERYTHING",
  "SET_MOOD": "Mood/SET_MOOD",
  "UPDATE_COMMENT": "Mood/UPDATE_COMMENT",
}
`;
